src/mount.ts

```diff
@@ -261,6 +264,33 @@ export function mount(
     to.appendChild(el)
   }
 
+  function slotToFunction(slot: Slot): Function {
+    if (typeof slot === 'object' && 'render' in slot && slot.render) {
+      return slot.render
+    }
+
+    if (typeof slot === 'function') {
+      return slot
+    }
+
+    if (typeof slot === 'object') {
+      return () => slot
+    }
```

Comment:
nit: I would write a single ` if (typeof slot === 'object')` and then deal with two cases inside.